src/lava/proc/atrlif/models.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (C) 2021-23 Intel Corporation
```

Comment:
Could you please update the copyright years for new files to just 2024 and for modified files to xx-2024 (for the Intel Corp line and your own line).

Thanks!